cmd/bosun/expr/expr.go

```diff
@@ -98,6 +99,8 @@ func (e *Expr) ExecuteState(s *State, T miniprofiler.Timer) (r *Results, queries
 	defer errRecover(&err)
 	if T == nil {
 		T = new(miniprofiler.Profile)
+	} else {
```

Comment:
this seems like a really odd condition for enabling computations. What is the logic there?